near/contracts/bridge/src/lib.rs

```diff
@@ -848,18 +982,29 @@ impl FastBridge {
         amount: U128,
         recipient_id: AccountId,
     ) -> U128 {
-        if is_promise_success() {
+        let mut transferred_amount = U128(0);
+
+        if let Some(result) = promise_result_as_success() {
+            transferred_amount = if result.is_empty() {
```

Comment:
why if `result.is_empty()` we transferred `amount` tokens?

Reply:
Because the `ft_transfer` doesn't return values